lib/runtime/runtime.go

```diff
@@ -34,6 +34,7 @@ type Ctx struct {
 	storage   Storage
 	allocator *FreeingBumpHeapAllocator
 	keystore  *keystore.GenericKeystore
+	validator int32
```

Comment:
this might make more sense as a bool? then convert it to int32 inside ext_is_validator